Client/Frontend/Browser/BrowserScrollController.swift

```diff
@@ -106,8 +106,21 @@ class BrowserScrollingController: NSObject {
 }
 
 private extension BrowserScrollingController {
+    func browserIsLoading() -> Bool {
+        if browser == nil {
+            return true
+        }
+
+        if let loading = browser?.loading where loading {
+            return true
+        }
+        return false
```

Comment:
You could reduce this method to a single line if you wanted: `return browser?.loading ?? true'